src/Menu/Menu.css

```diff
@@ -0,0 +1,57 @@
+.textbox {
+    margin-bottom: 20px;
+}
+
+.itemList {
+    list-style: none;
+    width: 280px;
+}
+.item {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border: 0.5px solid rgb(200, 196, 196);
+    padding: 5px;
+}
+
+.partyList {
+    list-style: none;
+}
+
+.sprite {
+    width: 20px;
+    margin-right: 5px;
+}
+
+.partyItem {
+    display: flex;
+    align-items: center;
+    margin: 5px;
+}
+.back {
+    float: right;
+}
+
+.movesMenu {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+}
+
+.attack {
+    display: flex;
+    flex-basis: calc(50% - 30px);
+    justify-content: center;
+    flex-direction: column;
+    margin: 10px;
+}
+
+.mainMenu {
+    max-width: 250px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+.menuButtons {
```

Comment:
you could remove this class from all the menu buttons and write
`.mainMenu > button { margin: 10px }`
instead if you wanted